src/test/test_integration.py

```diff
@@ -155,6 +155,10 @@ def test_import_nonexistent_existing(self):
         obj_doc2 = _wait_for_re_doc('ws_object', '41347:6')
         self.assertEqual(obj_doc1['_rev'], obj_doc2['_rev'])
 
+    def test_reload_aliases(self):
+        """test the RELOAD_ELASTIC_ALIASES event. TODO: fill out test."""
+        _produce({'evtype': "RELOAD_ELASTIC_ALIASES", 'wsid': 1, 'objid': 1, 'ver': 1})
```

Comment:
Would be good to have a test here. Maybe you can iterate over the aliases from the config and compare them to the response from `/_cat/aliases`?
Did you run the server on your laptop and verify manually that the aliases look good?